src/models/Genre/getAll.ts

```diff
@@ -1,13 +1,24 @@
 import { DataWithPaginationResponse } from '../../types/apiResponse';
-import { PaginationOptions } from '../../types/pagination';
 import { cdaClient, DEFAULT_CONTENTFUL_LIMIT } from '../../utils/contentful';
+import { PaginationOptions } from '../../types/pagination';
 import { Genre } from './GenreModel';
 import parseGenre from './parseGenre';
 
+type GetAllOptions = PaginationOptions & {
+	include?: 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
+	query?: {
+		id?: string;
+		title?: string;
+	};
+};
+
 export default async function getAll({
 	page = 1,
 	limit = DEFAULT_CONTENTFUL_LIMIT,
-}: PaginationOptions = {}): Promise<DataWithPaginationResponse<Genre>> {
+	include = 0,
```

Comment:
What is this and what are you doing with it? I don't love the type assigned so need an explanation.